gcp_variant_transforms/testing/data/misc/partition_config1.yaml

```diff
@@ -0,0 +1,37 @@
+-  partition:
+     partition_name: "chr01_part1"
+     regions:
+       - "chr1:0-1,000,000"
+-  partition:
+     partition_name: "chr01_part2"
+     regions:
+       - "chr1:1,000,000-2,000,000"
+-  partition:
+     partition_name: "chr01_part3"
+     regions:
+       - "chr1:2,000,000-999,999,999"
+-  partition:
+     partition_name: "chrom02"
+     regions:
+       - "chr2"
+       - "ch2"
```

Comment:
please avoid using "ch2" and "c2" even in tests as they're not 'standard' and can confuse people looking at this test :) You can choose one of the real chromosome names from hg19.